qiskit/ignis/verification/topological_codes/fitters.py

```diff
@@ -373,19 +395,158 @@ def matching(self, string):
 
         return logical_string
 
-    def get_logical_prob(self, results, algorithm='matching'):
+    def Nearest_Cluster(self, Cluster, graph, target):
+        """Find the nearest cluster to the target cluster.
+
+        Args:
+            Cluster (dict): Dictionary that contains clusters in the
+            Error graph and the nodes in it.
+
+            graph (retworkx.PyGraph):Error graph in which the
+            nearest cluster and the node will be searched.
+
+            target (int,int,int) : target cluster for which nearest
+            cluster is being searched.
+
+        Returns:
+            list: [nearest_outside_node, nearest_cluster]
+            nearest_outside_node : nearest node to the target node
+            which doesn't belong to the same cluster.
+            nearest_cluster : cluster to which nearest outside node
+            belongs.
         """
+        Cluster_Graph = rx.PyGraph()
```

Comment:
I don't expect this will pass the linting rules for variable naming, normally you'd use `cluster_graph` without capital letters.

Reply:
Thanks again. I will check it with lint again. It passes when I checked it but will do it again for sure.

Reply:
Changed